tests/acceptance/api_test.go

```diff
@@ -2071,6 +2074,7 @@ func parseResponseArrayJSON(t *testing.T, resp *apiResponse) []map[string]interf
 	// Go doesn't allow you to cast directly to []map[string]interface{}
 	// so we need to mess around here a bit.
 	// This might be relevant: https://stackoverflow.com/questions/38579485/golang-convert-slices-into-map
+	require.IsTypef(t, []interface{}{}, resp.json, "Expected JSON array response, got %v", resp.json)
```

Comment:
Previous, tests would panic if we got an error response instead of a JSON list. Add a check here to fail more gracefully.

Reply:
Thanks.  This is one of the things that was annoying when I was discussing the different response structures.  If you expect a list but get an error you have to adjust how you parse the output.  This should help.